src/js/utils/MesosStateUtil.js

```diff
@@ -110,14 +110,15 @@ const MesosStateUtil = {
     }, {});
   },
 
-  getTasksFromVirtualNetworkName({ tasks = [] } = {}, overlayName) {
+  getRunningTasksFromVirtualNetworkName({ tasks = [] } = {}, overlayName) {
     return tasks.filter(function(task) {
       const appPath = "container.network_infos.0.name";
       const podPath = "statuses.0.container_status.network_infos.0.name";
 
       return (
-        Util.findNestedPropertyInObject(task, appPath) === overlayName ||
-        Util.findNestedPropertyInObject(task, podPath) === overlayName
+        Util.findNestedPropertyInObject(task, "state") === "TASK_RUNNING" &&
```

Comment:
Anything else we want to display? `TASK_STARTING` maybe?

Reply:
Concerning "running" tasks do we maybe mean "active" tasks. As you mention there are more that one states where it is considered "running" or "active".

In the TasksStatusFilter.js to get the active tasks I've use the following method `TaskStates[task.state].stateTypes.includes("active");`

Maybe we have to use the same logic to show the "active" tasks. I may have confused what is considered "running" or "active", but looks like it is the same.